kylin-spark-project/kylin-spark-query/src/main/scala/org/apache/spark/sql/SparderContext.scala

```diff
@@ -177,7 +172,7 @@ object SparderContext extends Logging {
                   .getContextClassLoader
                   .toString)
               initMonitorEnv()
-              System.getProperty("spark.local") match {
+              master match {
                 case "true" =>
```

Comment:
The value of 'master' is 'local' or 'yarn-client', please modify 'case "true"' to 
```case mode: String if mode.startsWith("local") =>```